y/y.go

```diff
@@ -41,6 +41,9 @@ var (
 	// compression algorithm is being used for compression. ZSTD cannot work
 	// without CGO.
 	ErrZstdCgo = errors.New("zstd compression requires building badger with cgo enabled")
+
+	// ErrDoAfterFinish indicates a pointless Do call after Finish
+	ErrDoAfterFinish = errors.New("Do has no any point after Finish")
```

Comment:
Looks good to me. 🥳 
Can you just rephrase this to something like "Do should not be called after Finish"?
@jarifibrahim what do you say?

Reply:
Done.